build/.editorconfig

```diff
@@ -0,0 +1,11 @@
+[*.cs]
```

Comment:
This helps get nuke build scripts formatting well.
(though I'm not convinced its working 100% right now - I had issues adding a new target; it wanted to format it oddly)